admin/js/swonpage-navigation.js

```diff
@@ -157,12 +91,22 @@
         pageNameElement.innerHTML = '<b style="display:block;padding:5px;padding-bottom:10px">Page name:</b><input id="sw-onpage-createpage-parent" type="hidden" value="' + folder + '" /><input id="sw-onpage-createpage-name" type="text" style="font-size:20px" />';
         dialogContent.appendChild(pageNameElement);
 
+        var pageNameElement = document.createElement('div');
+        pageNameElement.innerHTML = '<b style="display:block;padding:5px;padding-bottom:10px">Page url:</b><span style="font-size:20px">' + folder + '<input id="sw-onpage-createpage-url" type="text" style="font-size:20px" pattern="[A-Za-z0-9]+" />';
+        dialogContent.appendChild(pageNameElement);
+
         var templates = document.createElement('div');
         templates.innerHTML = '<b style="display:block;padding:5px;padding-bottom:10px;padding-top:30px">Choose template to use:</b>loading page layout...';
         dialogContent.appendChild(templates);
 
         document.getElementsByTagName('body')[0].appendChild(dialog);
 
+        var closeBtn = document.getElementsByClassName('sw-onpage-navigation-item-close')[0];
+        closeBtn.addEventListener('click', function (e) {
```

Comment:
Since you're not using the (e), you can just put `dialog.remove` directly as the callback